lesson6.html

```diff
@@ -67,7 +68,7 @@ <h3>Dictionaries</h3>
 
             <div class="fragment">
               <pre><code class="python" data-trim>
-                # We can define the dictionary in one line too
+                # We can define a dictionary in one line too
                 eng_to_sp = {'one': 'uno', 'two': 'dos', 'three': 'tres'}
```

Comment:
Maybe this should be changed to use the same `eng_dict` example?

Reply:
I kept this so we have a variety of examples so the concept hopefully makes more sense, but I can see your point. Maybe it can get overwhelming with so many different ones. What do you guys think @agutteridge @chi-lp?

Reply:
I would leave the same one, I think it would be easier to understand

Reply:
agreed, same dictionary-dictionary example would be good, and then we expand the concept later with the shopping cart :)

Reply:
26bb9484680e000a6a528e01542a542c1f9717df